src/javatests/com/google/cloud/bigquery/dwhassessment/hooks/logger/DatePartitionedRecordsWriterFactoryTest.java

```diff
@@ -47,6 +46,8 @@ public class DatePartitionedRecordsWriterFactoryTest {
   @Rule public TemporaryFolder folder = new TemporaryFolder();
 
   private static final String TEST_ID = "a665f132";
+  private static final long DEFAULT_TIMESTAMP =
+      parseDateTime("2022-12-25T12:00:00.00Z").toEpochMilli();
```

Comment:
Optional: I would recommend changing the type of `DEFAULT_TIMESTAMP` to `Instant` and convert it to `long` at the last possible moment, when this is needed, instead of the other way around (i.e. the value is converted to epoch milli here, but then it is converted back to `Instant` at some of the usage sites).

Reply:
Done